### ISSUE TITLE
Deprecated: ServiceLocatorAwareInterface is deprecated and will be removed in version 3.0

### ISSUE THREAD

**Issue by ghost** [Lansoweb/LosDomain#2]
Hi i have this warning 

to avoid i put in my Domain.php

  public function __construct($domain = null, $config)
    {
        $this->config=$config;
        if ($domain === null) {
            $domain = self::getDomain();
        }
        $this->setDomain($domain);
    }
and in my Module.php 

'LosDomain\Service\Domain' => function (ServiceLocatorInterface $sl) {
                    $config = $sl->get('Configuration');
                    $domain = new DomainService(null,$config);

```
                return $domain;
            },
```

**Comment by Lansoweb** [comment]
@mpozzo You are right. I will update this module for zf3 components soon. Thanks!

**Comment by pdrosos** [comment]
@Lansoweb any news about the ZF3 compatibility? Thanks!

**Comment by Lansoweb** [comment]
Sorry for the delay. I just updated the code in the develop branch. Could you please try this branch in your project. Just specify in the composer.json:
```
"los/losdomain" : "dev-develop"
```

Thanks!

**Comment by pdrosos** [comment]
@Lansoweb tried the dev branch and seems to work. Thanks!

**Comment by Lansoweb** [comment]
@pdrosos Thanks! Will generate a new version now.

**Comment by Lansoweb** [comment]
Released [1.0.0](https://github.com/Lansoweb/LosDomain/releases/tag/1.0.0) with ZF3 support. Thanks for the help!
